lib/Reader/reader.cpp

```diff
@@ -6994,6 +7001,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
     case ReaderBaseNS::CEE_LDC_I4_8:
     case ReaderBaseNS::CEE_LDC_I4_M1:
     LOAD_CONSTANT:
+      nop();
```

Comment:
I saw this with breaks too, which is where that nop for branches came from. I'm not sure how I would go about doing the check of if the last instruction in the current block doesn't have the stack empty debug info.